### ISSUE TITLE
Active Screen and App suspension icon indicators

### ISSUE THREAD

**Issue by bigbrotherisstillwatching** [kugiigi/jerk-packages#39]
The Active screen and App suspension icon indicators in the Top Bar are hidden behind the Launcher when NOT in Desktop Mode. I don't know if this is somehow changeable/fixable, in Desktop Mode the Launcher Bar is shorter and does not overlap the Top Bar.

**Comment by kugiigi** [comment]
I'm not sure this is worth doing since the Launcher always auto-hides in Staged mode, unless we add an option to not auto-hide the Launcher which may be useful on tablets.

**Comment by bigbrotherisstillwatching** [comment]
You're right as for the App suspension icon indicator since it's only visible when apps are open. But the Active Screen icon indicator is also shown when no apps are open. Therefore it's worth, i think, especially when you've planned to implement an option to not auto-hide the launcher.

**Comment by kugiigi** [comment]
I wanted to fix this but at the moment the top bar is really crowded especially when there's a display cutout.
I'm not sure how to handle this. I think the top bar needs a redesign 😅
